### PR TITLE
Feature module user

### PR DESCRIPTION
**Pull request by silvermiguel13**
## Summary

[Change!] Describe your feature, problems you had, notes, improvements and others.

Created the model and migrations to users

## Known Issues

-

## Trello Card

https://trello.com/c/MeoJM1ov

### CODE REVIEW

File: app/models/user.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define('users', {
+    ID: { type: DataTypes.INTEGER, unique: true, primaryKey: true, allowNull: false },
+    name: { type: DataTypes.STRING, allowNull: false },
+    lastName: { type: DataTypes.STRING, allowNull: false },
+    gmail: { type: DataTypes.STRING, unique: true, allowNull: false },
+    password: { type: DataTypes.STRING, allowNull: false }
+  });
```
Comment (shernanm) [inline app/models/user.js:8]: Check the `underscored` property here to transform the camelCase model properties to snake_case.

File: app/models/user.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define('users', {
+    ID: { type: DataTypes.INTEGER, unique: true, primaryKey: true, allowNull: false },
+    name: { type: DataTypes.STRING, allowNull: false },
+    lastName: { type: DataTypes.STRING, allowNull: false },
+    gmail: { type: DataTypes.STRING, unique: true, allowNull: false },
```
Comment (shernanm) [inline app/models/user.js:6]: Why gmail? It seems to me that email is more generic.

File: app/models/user.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define('users', {
+    ID: { type: DataTypes.INTEGER, unique: true, primaryKey: true, allowNull: false },
```
Comment (shernanm) [inline app/models/user.js:3]: Columns should be lowercase.

File: migrations/migrations/20210831184614-users.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    await queryInterface.createTable('users', {
+      ID: { type: Sequelize.INTEGER, unique: true, autoIncrement: true, allowNull: false },
+      name: { type: Sequelize.STRING, allowNull: false },
+      lastName: { type: Sequelize.STRING, allowNull: false },
+      gmail: { type: Sequelize.STRING, unique: true, allowNull: false },
+      password: { type: Sequelize.STRING, allowNull: false }
+    });
+  },
+
+  // eslint-disable-next-line no-unused-vars
```
Comment (shernanm) [inline migrations/migrations/20210831184614-users.js:14]: What do you use this rule for?

File: migrations/migrations/20210831184614-users.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('users', {
+      ID: { type: Sequelize.INTEGER, unique: true, autoIncrement: true, allowNull: false },
+      name: { type: Sequelize.STRING, allowNull: false },
+      lastName: { type: Sequelize.STRING, allowNull: false },
+      gmail: { type: Sequelize.STRING, unique: true, allowNull: false },
+      password: { type: Sequelize.STRING, allowNull: false }
+    });
```
Comment (shernanm) [inline migrations/migrations/20210831184614-users.js:11]: The `created_at` and `updated_at` properties are missing

**Review by shernanm — CHANGES_REQUESTED** [review]

File: app/models/user.js (hunk 2 above)
Comment (silvermiguel13) [reply app/models/user.js:6]: Sorry, I don't happened to my

**Review by shernanm — APPROVED** [review]

**Pull request merged**